atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -226,12 +228,27 @@ def manifest_list_entries_match(self, image, build_id):
             v2_digest = archive['extra']['docker']['digests'][v2_type]
             koji_archives_data[arch] = v2_digest
 
-        if koji_archives_data == manifest_list_data:
-            self.log.info('Deeper manifest list check verified v2 manifest references match')
-            return True
-        self.log.warning('Manifest list refs "%s" do not match koji archive refs "%s"',
-                         manifest_list_data, koji_archives_data)
-        return False
+        if not all(arch in koji_archives_data for arch in self.architectures):
```

Comment:
we will have to convert self.arches to that different format 
self.arches has x86, koji archive/manifest_list amd64